config/console.php

```diff
@@ -2,22 +2,40 @@
 
 declare(strict_types=1);
 
+use Yiisoft\Aliases\Aliases;
+use Yiisoft\Factory\Definition\Reference;
+use Yiisoft\View\View;
 use Yiisoft\Yii\Db\Migration\Informer\MigrationInformerInterface;
 use Yiisoft\Yii\Db\Migration\Informer\NullMigrationInformer;
+use Yiisoft\Yii\Db\Migration\Service\Generate\CreateService;
 use Yiisoft\Yii\Db\Migration\Service\MigrationService;
 
 /** @var array $params */
 
 return [
     MigrationService::class => [
         'class' => MigrationService::class,
-        'createNamespace()' => [fn () => $params['yiisoft/yii-db-migration']['createNamespace']],
-        'updateNamespace()' => [
-            fn () => $params['yiisoft/yii-db-migration']['updateNamespace'],
-        ],
+        'createNamespace()' => [$params['yiisoft/yii-db-migration']['createNamespace']],
+        'updateNamespace()' => [$params['yiisoft/yii-db-migration']['updateNamespace']],
```

Comment:
I think we should rename this method to "updateNamespace**s**" because it's very confusing now that "createNamespace" accepts a single value but "updateNamespace" is expecting array.

```suggestion
        'updateNamespaces()' => [$params['yiisoft/yii-db-migration']['updateNamespaces']],
```

Reply:
done.